manifests/init.pp

```diff
@@ -90,6 +90,7 @@
   $jvm_xmx                                                          = '1024m',
   $jvm_permgen                                                      = '256m',
   $jvm_optional                                                     = '-XX:-HeapDumpOnOutOfMemoryError',
+  $jvm_extra_args                                                   = '-XX:+PrintGCDateStamps'  # remove for openjdk 11
```

Comment:
I think this could be changed to `'-XX:+PrintGCDateStamps -XX:+ExplicitGCInvokesConcurrent'` so jira doesn't complain about explicit garbage collection.  See -

https://confluence.atlassian.com/jirakb/health-check-explicit-garbage-collection-976778100.html